### ISSUE TITLE
Enable ability to display and delete relations even if no longer in the allowed list or invalid

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/noctua-visual-pathway-editor#50]
Some existing models may use relations that are no longer available in the decision tree, e.g. 'positively regulates'.

We still want users to be able to delete these relations so they can replace them with the correct one.

In these cases, opening up the pop-up decision window will NOT default to any particular value.

See this model as a test to try to delete regulates between unc-13 and rad-26 activity nodes:

http://noctua-dev.berkeleybop.org/workbench/noctua-visual-pathway-editor/?model_id=gomodel%3A636039dd00000000

@pgaudet

**Comment by vanaukenk** [comment]
@tmushayahama 
Can we PR this to dev before tomorrow?
It'd be great to get this onto prod by the end of this week.
Thanks!

**Comment by tmushayahama** [comment]
yes, lemme PR soon. Added a warning text if the relationship is not supported

![image](https://user-images.githubusercontent.com/2273929/214709812-4406aaa1-37fb-493a-bb27-dbb490702d1b.png)

**Comment by vanaukenk** [comment]
Thanks @tmushayahama @kltm 
I just tested on noctua-dev and things are working fine.
I like the warning message; it helps to explain the tool behavior.
We can move forward with putting the pop-up window on production.

**Comment by thomaspd** [comment]
This has been fixed so I'm closing this issue.
